src/promptflow-evals/promptflow/evals/evaluators/_fluency/_fluency.py

```diff
@@ -7,14 +7,48 @@
 
 import numpy as np
 
-from promptflow.client import load_flow
-from promptflow.core import AzureOpenAIModelConfiguration
+from promptflow._utils.async_utils import async_run_allowing_running_loop
```

Comment:
Regarding this dependency can we add test in promptflow-core to make sure we cover this in case of change

Reply:
Sure, added.